### PR TITLE
Example program: Add indexing interval description & fix logging message

### PR DESCRIPTION
**Pull request by kevindiu**
### Description:
This PR contains the following changes:
1. Add indexing duration on the example program
2. Add comments about the indexing feature
3. Fix logging message on example program

### Environment:

- Golang Version: 1.14.3
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.11.5

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [x] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [x] I have added tests and benchmarks to cover my changes.
- [x] I have ensured all new and existing tests passed.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by vankichi** [comment]
@kevindiu @hlts2 
please tell me when you have done check wether this fix is correct or not via running this tutorial.

File: example/helm/values-scylla.yaml [hunk 1]
```diff
@@ -53,6 +53,13 @@ indexManager:
     requests:
       cpu: 100m
       memory: 30Mi
+  indexer:
+    # The limit duration of automatic indexing.
+    # This trigger the force indexing on each Vald Agent.
+    auto_index_duration_limit: 1m
+    # The check duration of automatic indexing.
+    # This trigger the indexing on each Vald Agnet if uncommitted index exceed limit.
+    auto_index_check_duration: 30s
```
Comment (hlts2) [inline example/helm/values-scylla.yaml:62]:
```suggestion
    auto_index_check_duration: 10s
```

**Pull request merged**